main/nodejs/APKBUILD

```diff
@@ -44,18 +44,17 @@
 pkgname=nodejs
 # Note: Update only to even-numbered versions (e.g. 6.y.z, 8.y.z)!
 # Odd-numbered versions are supported only for 9 months by upstream.
-pkgver=10.16.3
+pkgver=12.13.0
 pkgrel=0
 pkgdesc="JavaScript runtime built on V8 engine - LTS version"
 url="https://nodejs.org/"
 arch="all !mips64 !mips64el"
 license="MIT"
 depends="ca-certificates"
 depends_dev="libuv"
-# gold is needed for mksnapshot
-makedepends="$depends_dev python2 openssl-dev zlib-dev libuv-dev linux-headers
-	paxmark binutils-gold http-parser-dev ca-certificates c-ares-dev"
-subpackages="$pkgname-dev $pkgname-doc npm::noarch"
+makedepends="linux-headers python2 paxmark
+	zlib-dev libuv-dev openssl-dev c-ares-dev nghttp2-dev"
```

Comment:
Just to make sure, we can't use py3?

Reply:
I've just copied the current `APKBUILD` of https://github.com/alpinelinux/aports/blob/master/community/nodejs-current/APKBUILD since this is/was already upgraded for Node.js v10. AFAIK python2 is still the recommended way to build Node.js: https://github.com/nodejs/node/blob/master/BUILDING.md#note-about-python-2-and-python-3

Reply:
Well, if py2 is recommended but we can use py3 it'd be best if we used py3 since we want to get py2 out of 3.11 if possible.

Reply:
I've just tried some ways (`alias` and `ln`), but I it seems python2 is still required:
- https://cloud.drone.io/alpinelinux/aports/13105/2/1 (line 124)
- https://cloud.drone.io/alpinelinux/aports/13099/2/1 (line 122)

Reply:
Ah, that's unfortunate, but thanks for trying